src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -336,5 +392,123 @@ status:
   ready: false
 ```
 
+{{< /tab >}}
+{{< tab id="command-output-openstack" title="OpenStack">}}
+
+```yaml
+---
+apiVersion: v1
+data:
+  values: |
+    bastion:
+      flavor: a1.tiny
+      image: ""
+      rootVolume:
+        diskSize: 10
+        sourceUUID: 4b4bbad1-0eca-4937-9fe5-f5e767b36c5f
+    cloudConfig: cloud-config-giantswarm-2
+    cloudName: openstack
+    controlPlane:
+      diskSize: 50
+      machineFlavor: a1.small
+      replicas: 1
+    externalNetworkID: 0d2b02e2-6003-4c09-8457-604d0a963f7a
+    nodeCIDR: 10.6.0.0/24
+    nodeClasses:
+    - diskSize: 50
+      machineFlavor: a1.small
+      name: default
+    nodePools:
+    - class: default
+      name: default
+      replicas: 2
+    oidc:
+      enabled: true
+    organization: multi-project
+    rootVolume:
+      enabled: true
+      sourceUUID: c3cc570d-cfd5-4cdb-aa46-fa82a6e72ecb
+kind: ConfigMap
+metadata:
+  creationTimestamp: null
+  name: demo1-userconfig
+  namespace: org-multi-project
+---
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  labels:
+    app-operator.giantswarm.io/version: 0.0.0
+  name: demo1
+  namespace: org-multi-project
+spec:
+  catalog: giantswarm
+  config:
```

Comment:
Something from my `to-do` which I haven't written down yet:
We should not have these empty fields.

Reply:
You'll need to make these fields pointers then https://github.com/giantswarm/apiextensions-application/blob/main/api/v1alpha1/app_types.go#L68 (`omitempty` doesn't work for non-pointer structs) or else do some string-based post-processing of the YAML.

Reply:
Opened an issue in the Rocket board so we don't forget this.